cmds/bootcm4.c

```diff
@@ -0,0 +1,192 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * boot CM4 CPU i.MX RT117x
+ *
+ * Copyright 2022-2023 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "cmd.h"
+
+#include <hal/hal.h>
+#include <phfs/phfs.h>
+#include <lib/lib.h>
+
+
+#define CM4_BOOT_MEMSIZE 0x00040000u
+#define CM4_BOOT_ADDRESS 0x20200000u
+
+
+static struct {
+	unsigned int isImgLoaded;
+	addr_t cm4VtorOffset;
+} common = { 0 };
```

Comment:
You use variable `common` only in `cmd_butcm4()` function.
Define `isImgLoaded` and `cm4VtorOffset` as `local` in this function.